nubia/internal/typing/argparse.py

```diff
@@ -134,7 +134,7 @@ def register_command(argparse_parser, inspection):
             dest="_subcmd",
             help=_command.help,
             parser_class=create_subparser_class(subparser),
-            metavar="[subcommand]".format(_command.name),
+            metavar="[subcommand]",
```

Comment:
Does this change has any consequences? It seems a bit weird

Reply:
It seems a formatting fix, not a real code change.